### PR TITLE
Fixes #9752: Document stopping process before Postgres restore maintenance to avoid primary key constrain get corrupted

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/9752

### CODE REVIEW

File: src/reference/modules/installation/pages/upgrade/postgresql.adoc [hunk 1]
```diff
@@ -10,6 +10,24 @@ This simplified PostgreSQL documentation. If in doubt, please always refer to th
 This documentation is about Debian and derived distributions since it is usual to upgrade major versions with them.
 On RHEL and SLES, it is recommended to reinstall the system, this documentation is then not needed. If you need it, please ask.
 
+[CAUTION]
+
+====
+Before perform any upgrade, make sure to *disable those services*, by disabling rudder agent *first* to be
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/postgresql.adoc:16]:
```suggestion
Before performing any upgrade, make sure to *disable those services*, by disabling rudder agent *first* to be
```

File: src/reference/modules/installation/pages/upgrade/postgresql.adoc [hunk 2]
```diff
@@ -10,6 +10,24 @@ This simplified PostgreSQL documentation. If in doubt, please always refer to th
...
+sure that it will not enable rudder-relayd and rudder-jetty back
+
+1. `rudder agent disable`
+2. `systemctl stop rudder-relayd`
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/postgresql.adoc:20]:
```suggestion
2. `systemctl stop rudder-relayd rudder-jetty`
```

File: src/reference/modules/installation/pages/upgrade/postgresql.adoc [hunk 3]
```diff
@@ -10,6 +10,24 @@ This simplified PostgreSQL documentation. If in doubt, please always refer to th
...
+3. `systemctl stop rudder-jetty`
```
Comment (amousset) [inline src/reference/modules/installation/pages/upgrade/postgresql.adoc:21]:
```suggestion
```

**Comment by ElaadF** [comment]
Commit modified

File: src/reference/modules/installation/pages/upgrade/postgresql.adoc [hunk 4]
```diff
@@ -10,6 +10,23 @@ This simplified PostgreSQL documentation. If in doubt, please always refer to th
 This documentation is about Debian and derived distributions since it is usual to upgrade major versions with them.
 On RHEL and SLES, it is recommended to reinstall the system, this documentation is then not needed. If you need it, please ask.
 
+[CAUTION]
+
+====
+Before performing any upgrade, make sure to *disable those services*, by disabling rudder agent *first* to be
```
Comment (ncharles) [inline src/reference/modules/installation/pages/upgrade/postgresql.adoc:16]: you need an empty line before here

File: src/reference/modules/installation/pages/upgrade/postgresql.adoc [hunk 5]
```diff
@@ -10,6 +10,23 @@ This simplified PostgreSQL documentation. If in doubt, please always refer to th
...
+sure that it will not enable rudder-relayd and rudder-jetty back
+
+1. `rudder agent disable`
+2. `systemctl stop rudder-relayd rudder-jetty`
+
+Upgrading PostgreSQL without disabling these service can lead to inconsistencies and breaks some constraints on the database.
```
Comment (ncharles) [inline src/reference/modules/installation/pages/upgrade/postgresql.adoc:22]:
```suggestion
Upgrading PostgreSQL without disabling these service can lead to inconsistencies and might break constraints within the database schema.
```

**Review by ncharles — CHANGES_REQUESTED** [review]
Great content, i made some suggestion on wording

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR rebased

**Review by ncharles — APPROVED** [review]

**Pull request merged**